phases/1-programming.md

```diff
@@ -2,99 +2,78 @@
+    * Event driven
+    * Others...
+  * Connect Nodejs to Chrome debugger
+  * Read "Node.js 8 the Right Way"
+    * Part 1
+    * Chapters 1 & 2
+  * Exercises from "Exercises for Programmers"
+    * Read Chapter 1
+    * Pick exercises to try from Chapter 8 (Working with Files)
+      * Create Repo for each exercise
+      * Submit unsquashed PRs (Sparkbox standard commits) for review
+  * [Use Chrome for debugging](https://nodejs.org/en/docs/guides/debugging-getting-started/)
 
-* [Explain Shell][]
-* [From Zero to Hero with Dotfiles: Pt 1][]
-* [From Zero to Hero with Dotfiles: Pt 2][]
-* [40 Terminal Tips & Tricks][]
-* [Git Cheat Sheet][]
-* [Practical Object Oriented Programming in Ruby][] (“sb-bookshelf” in Dropbox)
-* [Learn the Command Line the Hard Way][]
-* [Learn Ruby the Hard Way][]
-* [Why's Poignant Guide to Ruby][]
-* [Apprenticeship Patterns][]
-* Practical Vim ("sb-bookshelf" in Dropbox)
-* [Treehouse][]
-* [Learn Git Branching][]: Learn Git visually and interactively
-* [Principles Wiki][]: Study software design principles
-* [Learning JavaScript Design Patterns][]
+* Week 3: Functional Programming
+  * Read Functional-Light JavaScript
+    * Chapters 1 & 8
+  * Read JavaScript Allonge
+    * Prelude
+  * "Exercises for Programmers"
+    * Pick exercises to try from Chapter 6 (Repetition)
+      * Create Repository for each exercise
+      * Submit unsquashed PRs (Sparkbox standard commits) for review
 
-[Getting Real]: http://gettingreal.37signals.com/
-[Passionate Programmer]: http://www.amazon.com/The-Passionate-Programmer-Remarkable-Development/dp/1934356344
-[Pragmatic Programmer]: https://www.amazon.com/Pragmatic-Programmer-Journeyman-Master/dp/020161622X
+* Week 4: Object Oriented Programming
+  * Read ThoughtWorks Anthology
+    * Chapter 6 (Object Calisthenics)
+  * Exercises from "Exercises for Programmers"
+    * Pick exercises to try
+      * Chapter 2 - Exercise 4: Mad Lib
+      * Chapter 4 - Exercise 23: Troubleshooting  Car Issues
+      * Create Repository for each exercise
+      * Submit unsquashed PRs (Sparkbox Standard Commits) for review
 
-[CodeSchool: Try Git]: http://www.codeschool.com/courses/try-git
-[CodeSchool: Git Real]: http://www.codeschool.com/courses/git-real
-[CodeSchool: Git Real 2]: https://www.codeschool.com/courses/git-real-2
-[More Git]: ../projects/proj-more-git.md
-[CodeSchool: Try Ruby]: http://tryruby.org/
-[Ruby koans]: http://rubykoans.com/
-[CodeSchool: Ruby Bits 1]: http://www.codeschool.com/courses/ruby-bits
-[CodeSchool: Ruby Bits 2]: http://www.codeschool.com/courses/ruby-bits-part-2
-[CodeSchool: Testing with RSpec]: https://www.codeschool.com/courses/testing-with-rspec
-[Triangle Kata - Ruby]: http://web.archive.org/web/20140119031248/http://onestepback.org/vital_testing/
-[Game of Life Kata - Ruby]: https://github.com/garora/TDD-Katas#game-of-life-
-[Ruby Warriors]: https://www.bloc.io/ruby-warrior/#/
+* Week 5: Test-Driven Development
+  * Read Nodejs the Right Way
```

Comment:
Should we try and link the articles here to the [SB bookshelf](https://www.dropbox.com/home/Sparkbox%20Bookshelf/Node.js%208%20the%20Right%20Way?preview=node-js-8-the-right-way_p1_0.pdf) on dropbox?